UnitTests/PayPalTests/PayPalClient_Tests.swift

```diff
@@ -24,6 +24,7 @@ class PayPalClient_Tests: XCTestCase {
         var capturedResult: Approval?
```

Comment:
General Swift organization puts properties above methods. We've started to document that in our Style guide [here](https://hackmd.io/kIyOnIFIRrKY4b5wQ0dUpw?both#Code-Organization).